jython/YetAnotherAutoTrain.py

```diff
@@ -17,6 +17,8 @@
 #         Add the ability to create custom extensions.
 # Author:  Dave Sand copyright (c) 2018 - 2020
 
+# The following help content is also available at https://jmri.org//help/en/html/scripthelp/yaat/YAAT.shtml
```

Comment:
I don't think this is a problem, but I noticed that there is two slashes between jmri.org and help in this link:
`https://jmri.org//help/en/html/scripthelp/yaat/YAAT.shtml`